dozent/dozent.py

```diff
@@ -13,26 +13,43 @@
     from downloader_tools import DownloaderTools
 
 CURRENT_FILE_PATH = Path(__file__)
-DEFAULT_DATA_DIRECTORY = CURRENT_FILE_PATH.parent.parent / 'data'
-TWITTER_ARCHIVE_STREAM_LINKS_PATH = CURRENT_FILE_PATH.parent / 'twitter-archive-stream-links.json'
+DEFAULT_DATA_DIRECTORY = CURRENT_FILE_PATH.parent.parent / "data"
+TWITTER_ARCHIVE_STREAM_LINKS_PATH = (
+    CURRENT_FILE_PATH.parent / "twitter-archive-stream-links.json"
+)
```

Comment:
Is this for following along with PEP8 guidelines? I think black has functionality for figuring that out

Reply:
Maybe. I need to fix Black for PRs